knowledge-base/actions/fossa-contrib/fossa-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: Fossa Action
+github-token:
+  action-input:
+    input: github-token
+    is-default: true
+  permissions:
```

Comment:
@sph4674  There is no need for adding permissions for KBs having only `content: read` permission, because it gets added at workflow level by default.